queries/elixir/highlights.scm

```diff
@@ -1,180 +1,154 @@
-(identifier) @variable
-
-; _unused variables
-(unused_identifier) @comment
-
-; __MODULE__ and friends
-(special_identifier) @constant.builtin
-
-(module) @type
-
-[(atom) (keyword)] @symbol
-
-(integer) @number
-(float) @float
-
-[(true) (false)] @boolean
-
-(nil) @constant.builtin
-
-(comment) @comment
-
-[
-  ","
-  "."
-] @punctuation.delimiter
+(ERROR) @error
 
 [
   "("
   ")"
+  "<<"
+  ">>"
   "["
   "]"
   "{"
   "}"
-  "<<"
-  ">>"
 ] @punctuation.bracket
 
-(interpolation
- "#{" @punctuation.special
- "}" @punctuation.special) @none
-
 [
-  (heredoc_content)
-  (sigil_content)
-  (string_content)
-  (string_end)
-  (string_start)
-] @string
+  ","
+  "->"
+  "."
+] @punctuation.delimiter
 
 [
-  (heredoc_end)
-  (heredoc_start)
-  (sigil_end)
   (sigil_start)
-] @string.special
+  (sigil_end)
+  (heredoc_start)
+  (heredoc_end)
+] @punctuation.special
 
-(escape_sequence) @string.escape
+(interpolation "#{" @punctuation.special "}" @punctuation.special)
 
 [
   "after"
+  "and" 
+  "catch"
   "do"
+  "else"
   "end"
-] @keyword
-
-[
-  "and"
+  "fn"
   "in"
-  "not"
   "not in"
+  "not"
   "or"
-] @keyword.operator
+  "rescue"
+  "when" 
+] @keyword
+
+[
+  (comment) 
+  (unused_identifier)
+] @comment
+
+[
+  (heredoc_content)
+  (sigil_content)
+  (string)
+] @string
 
-; Call to a local function
-(call (function_identifier) @method)
+; __MODULE__ and friends
+(special_identifier) @constant.builtin
+
+(map "%{" @constructor "}" @constructor)
+
+(struct "%" @constructor "{" @constructor "}" @constructor)
+
+(binary_op operator: _ @operator)
```

Comment:
We tend to put parens around wildcard nodes :
```suggestion
(binary_op operator: (_) @operator)
```

Reply:
Hmm, I tried your suggestion and that broke highlighting for unary and binary operators 😦 Could we move forward with this as-is for now and revisit this later?

Reply:
`_` matches both named and anonymous nodes, `(_)` matches named nodes only.

Reply:
That would explain why the highlights broke with `(_)`.  Based on what I see in the playground, `operator` is an anonymous Elixir node that you can reference via a field.